docs/data/toolpad/core/introduction/tutorial.md

```diff
@@ -1,3 +1,149 @@
-# Tutorial
+---
+title: Tutorial
+---
+
+# Toolpad Core - Tutorial
 
 <p class="description">Tutorial</p>
+
+## Steps to follow
+
+1. Run:
+
+   ```bash
+   yarn create toolpad-app --tutorial my-first-project
+   ```
+
+   This will prompt for a project name and create a basic project for this tutorial
+
+2. Install the packages necessary for this example:
+
+   ```bash
+   yarn add -S @mui/x-data-grid @mui/x-charts @mui/toolpad-data-csv
```

Comment:
Would it make sense to already include them in the package.json of the tutorial template?